docs/CSBR.md

```diff
@@ -1115,31 +1707,31 @@ Log records MUST include the following elements:
+   3. Any attempt to delete or modify timestamp logs,
+   4. Security events, including:
+      1. Successful and unsuccessful Timestamp Authority access attempts;
+      2. Timestamp Authority server actions performed;
+      3. Security profile changes;
+      4. System crashes and other anomalies; and
+      5. Firewall and router activities;
+   5. Revocation of a timestamp certificate,
+   6. Major changes to the timestamp server's time, and
+   7. System startup and shutdown.
 
 ### 5.4.2 Frequency of processing log
 
 ### 5.4.3 Retention period for audit log
 
 The CA, Delegated Third Parties, and Timestamp Authority MUST retain, for at least two (2) years:
 
-1.	CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1.1](#5411-types-of-events-recorded-for-cas))(1) after the later occurrence of:
-   a.	the destruction of the CA Private Key; or
-   b.	the revocation or expiration of the final CA Certificate in that set of Certificates that have an X.509v3 basicConstraints extension with the cA field set to true and which share a common Public Key corresponding to the CA Private Key;
-2.	Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1.2](#5412-types-of-events-recorded-for-timestamp-authorities))(2) after the revocation or expiration of the Subscriber Certificate;
-3.	Timestamp Authority data records (as set forth in [Section 5.4.1.2](#5412-types-of-events-recorded-for-timestamp-authorities)) after the revocation or renewal of the Timestamp Certificate Private Key (as set forth in [Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods));
-4.	Any security event records (as set forth in [Section 5.4.1.1](#5412-types-of-events-recorded-for-timestamp-authorities)(3) and for Timestamp Authority security event records set forth in [Section 5.4.1.2](#5412-types-of-events-recorded-for-timestamp-authorities)(3)) after the event occurred
+1. CA certificate and key lifecycle management event records (as set forth in [Section 5.4.1.1](#5411-types-of-events-recorded-for-cas))(1) after the later occurrence of:
+   1. the destruction of the CA Private Key; or
+   2. the revocation or expiration of the final CA Certificate in that set of Certificates that have an X.509v3 basicConstraints extension with the cA field set to true and which share a common Public Key corresponding to the CA Private Key;
+2. Subscriber Certificate lifecycle management event records (as set forth in [Section 5.4.1.2](#5412-types-of-events-recorded-for-timestamp-authorities))(2) after the revocation or expiration of the Subscriber Certificate;
+3. Timestamp Authority data records (as set forth in [Section 5.4.1.2](#5412-types-of-events-recorded-for-timestamp-authorities)) after the revocation or renewal of the Timestamp Certificate private key (as set forth in [Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods));

```

Comment:
Why was "Private Key" changed to "private key"? The Defined Term seems ok to use here.